README.md

```diff
@@ -18,7 +18,7 @@ Kubernetes is:
 * **extensible**: modular, pluggable, hookable, composable
 * **self-healing**: auto-placement, auto-restart, auto-replication
 
-Kubernetes builds upon a decade and a half of experience at Google running production workloads at scale, combined with best-of-breed ideas and practices from the community.
+Kubernetes builds upon a [decade and a half of experience at Google running production workloads at scale](https://research.google.com/pubs/pub43438.html), combined with best-of-breed ideas and practices from the community.
```

Comment:
That text is copied all over. I could fix it, but I'm not sure it's worth it right now. I'm focused on content.